tests/test_objective_funs.py

```diff
@@ -1249,6 +1250,148 @@ def test(
             coils_fixed=True,
         )
 
+    @pytest.mark.unit
+    def test_plasma_coil_max_distance(self):
```

Comment:
in these cases, is the max also equal to the min?

Might be worth to try shifting one of these slightly outward so that there is an asymmetry introduced in the problem, so that you can check the max stuff more explicitly